package.json

```diff
@@ -27,6 +27,7 @@
     "@angular/platform-server": "16.2.1",
     "@angular/router": "16.2.1",
     "@cdktf/provider-aws": "14.0.2",
+    "@ngbracket/ngx-layout": "^17.0.0",
```

Comment:
- Pin version
- Should we select the same version as the Angular version, though it does not seem to be a package from the angular team?